### PR TITLE
Feature/open5gs k8s ue count

### PR DESCRIPTION
**Pull request by bjoern-r**
Feature request to automaticallyprovision a varaible numer of UEs into the UDM of open5gs.
Same syntax as used in open5gs_vm component.

### CODE REVIEW

**Comment by bjoern-r** [comment]
please test as berlin platform is currently on benchmark so i'm not 100% sure about the helm syntax and cannot deploy

**Review by alvarocurt — DISMISSED** [review]
Look nice, but I found a potential problem for future deployments:
- mnc can be 2 or 3 digits, so we can't assume msin is always 10 digits long. An extra check must be done so that the msin is 9 or 10 digits long to not break the 15 digit requirement.

**Comment by bjoern-r** [comment]
you are right, however the script before the change did also not take the mnc length into account.

File: open5gs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 ### Changed
```
Comment (jmaciasportela) [inline open5gs/changelog.md:4]: Please @bjoern-r increase the version number of the component into the changelog

**Review by jmaciasportela — CHANGES_REQUESTED** [review]

**Review by jmaciasportela — APPROVED** [review]

**Pull request merged**